GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs

```diff
@@ -170,12 +171,12 @@ public void SaveSplitterPositions()
             }
         }
 
-        public event GitModuleChangedEventHandler GitModuleChanged;
+        public event EventHandler<GitModuleEventArgs> GitModuleChanged;
 
-        public virtual void OnModuleChanged(GitModule aModule)
+        public virtual void OnModuleChanged(object sender, GitModuleEventArgs e)
         {
             if (GitModuleChanged != null)
```

Comment:
This should be assigned to a temporary variable, to ensure it is not affected by code operating on other threads.

```csharp
var t = GitModuleChanged;
if (t != null)
  t(this, e);
```

In C# 6, this is simplified dramatically to this:

```csharp
GitModuleChanged?.Invoke(this, e);
```